io_ogre/ui/export.py

```diff
@@ -244,137 +257,12 @@ def execute(self, context):
         name="copy shader programs",
         description="when using script inheritance copy the source shader programs to the output path",
         default=config.get('COPY_SHADER_PROGRAMS'))
-
```

Comment:
This commit has duplicated the config values. https://github.com/OGRECave/blender2ogre/commit/914fbced9fbe9d6baa27a0b33029a7f68cc0b1bf#diff-149842062e0cc714c18be472974455ddL138